scripts/talents/generate-talents.ts

```diff
@@ -40,7 +40,7 @@ const classes: { [classId: number]: { name: string; baseMaxResource: number } }
 
 async function generateTalents() {
   const talents: ITalentTree[] = await readJsonFromUrl(TALENT_DATA_URL);
-  const spellpower: ISpellpower[] = csvToObject(await readCsvFromUrl(SPELLPOWER_DATA_URL));
+  const spellpower: ISpellpower[] = csvToObject(await readCsvFromFile(SPELLPOWER_DATA_FILE));
```

Comment:
Since `readCsvFromFile` uses `readFileSync`, you don't need the `await` here, though it's not breaking anything.